docker/build-tools/Dockerfile

```diff
@@ -42,7 +42,7 @@ ENV COSIGN_VERSION=v2.1.1
 ENV CRANE_VERSION=v0.15.2
 ENV GCLOUD_VERSION=439.0.0
 ENV GCR_AUTH_VERSION=2.1.6
-ENV GH_VERSION=2.32.0
+ENV GH_VERSION=2.43.1
```

Comment:
Debian install for 2.32.0 wasn't being found. Perhaps we need to change this to rebuild (?).